crates/ruff_server/src/server/api/requests/execute_command.rs

```diff
@@ -19,6 +20,11 @@ struct Argument {
     version: DocumentVersion,
 }
 
+#[derive(Default, Deserialize)]
+struct DebugCommandArgument {
+    uri: Option<types::Url>,
```

Comment:
Could we document the `uri` parameter? The name doesn't give any information (nor the struct name) what the uri points to

Reply:
I'll replace `DebugCommandArgument` with [`TextDocumentIdentifier`](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocumentIdentifier) because that's what we want here.

Reply:
Looking at how `rust-analyzer` accepts parameters, it seems useful to have a consistent structure similar to how requests would accept params, so this could be updated to:
```
{ textDocument?: TextDocumentIdentifier }
```
For example:
```
{
    textDocument: {
        uri: "file:///Users/dhruv/dotfiles/config/nvim/lua/dm/lsp/extensions/rust_analyzer.lua"
    }
}
```